.gitignore

```diff
@@ -19,3 +19,5 @@ venv.bak/
 
 # MacOS
 .DS_Store
+
+install_id
```

Comment:
Is this accidental?

Reply:
oh yeah forgot to take it out, it was a couple of weeks ago when it kept being created